### PR TITLE
enable auto-approve and auto-merge for pull bot

### PR DESCRIPTION
**Pull request by anshbansal**
## Checklist

- [ ] The PR conforms to DataHub's [Contributing Guideline](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md) (particularly [Commit Message Format](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md#commit-message-format))
- [ ] Links to related issues (if applicable)
- [ ] Tests for the changes have been added/updated (if applicable)
- [ ] Docs related to the changes have been added/updated (if applicable). If a new feature has been added a Usage Guide has been added for the same.
- [ ] For any breaking change/potential downtime/deprecation/big changes an entry has been made in [Updating DataHub](https://github.com/datahub-project/datahub/blob/master/docs/how/updating-datahub.md)

### CODE REVIEW

**Comment by anshbansal** [comment]
@treff7es Whenever a change happens in datahub-project/datahub there is a bot that creates a PR which you might have noticed https://github.com/acryldata/datahub/pulls?q=is%3Apr+is%3Aclosed+label%3A%22%3Aarrow_heading_down%3A+pull%22

I am trying to auto-merge the PR because it is just doing a sync between the upstream and this repository. It needs an approval and someone needs to go in and click this "Enable auto-merge" for the PR to be auto-merged
image

Basically I am trying to automatically merge from upstream to this repository.

**Review by treff7es — APPROVED** [review]
